rudder-web/src/main/scala/com/normation/rudder/web/snippet/HomePage.scala

```diff
@@ -182,76 +187,72 @@ class HomePage extends Loggable {
        */
 
       val complianceByNode : List[Double] = reportsByNode.values.map { r =>
-        if(r.pending == r.total) -1 else  r.complianceWithoutPending
+        if(r.pending == r.total) -1
+        else if(r.reportsDisabled == r.total) -2
```

Comment:
this looks really like "magic numbers" and should be treated as such
plus, i'd really like { } around the ifs